rllib/utils/schedules/piecewise_schedule.py

```diff
@@ -12,7 +13,7 @@ def _linear_interpolation(left, right, alpha):
     return left + alpha * (right - left)
 
 
-@PublicAPI
+@DeveloperAPI
```

Comment:
We might think about a sinusoidal schedule - I see this more often in the papers now.